tests/dragonfly/cluster_test.py

```diff
@@ -123,6 +123,9 @@ async def test_cluster_slots_in_replicas(df_local_factory):
         ip="127.0.0.1", port=master.port, answer=res[0], rep_ip="127.0.0.1", rep_port=replica.port
     )
 
+    await c_master.close()
```

Comment:
this works but seems like code duplication.
why not use `await disconnect_clients` call like in other places?

Reply:
Didn't know about it thx